Permanent/Modules/EditMetadata/Views/MetadataEditFileNames.swift

```diff
@@ -33,19 +35,109 @@ struct MetadataEditFileNames: View {
                     CustomSegmentedControl(selectedItem: $selectedItem, items: menuItems)
                     Spacer()
                 }
-                .padding()
+                .padding(.top, 20)
+                .padding(.horizontal, 24)
                 .navigationBarTitle("Edit file names", displayMode: .inline)
                 .navigationBarItems(leading: Image("editFilenames")
                     .renderingMode(.template)
                     .foregroundColor(.white)
                     .frame(width: 24, height: 24))
             }
+            .padding(.bottom, 85)
+            VStack {
+                Spacer()
+                previewSection
+                    .padding(.horizontal, 10)
+                bottomButtons
+                    .padding(.horizontal, 10)
+            }
+        }
+    }
+    
+    var previewSection: some View {
+        VStack {
+            HStack {
+                Text("Preview".uppercased())
+                    .textStyle(SmallXXXXXSemiBoldTextStyle())
+                    .foregroundColor(.lightGray)
+                Rectangle()
+                    .foregroundColor(.clear)
+                    .frame(width: 286, height: 1)
+                    .background(Color.lightGray)
+            }
+            HStack(spacing: 16) {
+                Rectangle()
+                .foregroundColor(.clear)
+                .frame(width: 40, height: 40)
+                .background(
+                    WebImage(url: URL(string: viewModel.imagePreviewURL))
+                .resizable()
+                .aspectRatio(contentMode: .fill)
+                .frame(width: 40, height: 40)
+                .clipped()
+                )
+                VStack(alignment: .leading) {
+                    Text(viewModel.fileNamePreview ?? "")
+                        .foregroundColor(.middleGray)
+                        .textStyle(SmallRegularTextStyle())
+                        .multilineTextAlignment(.leading)
+                    Text(viewModel.fileSizePreview ?? "")
+                        .textStyle(SmallXXXRegularTextStyle())
+                        .multilineTextAlignment(.leading)
+                        .foregroundColor(.gray)
+                }
+            }
+            .frame(maxWidth: .infinity, alignment: .leading)
+            .padding()
+        }
+    }
+    
+    var bottomButtons: some View {
+        HStack {
+            Button {
+                presentationMode.wrappedValue.dismiss()
+            } label: {
+                ZStack {
+                    Rectangle()
```

Comment:
The best approach would be to create a button style instead as a viewModifier. Take as an example the TextStyle view modifier.